### DISCUSSION TITLE
About the precise definitions of little group and Herring little group.

### DISCUSSION THREAD

**Discussion by hongyi-zhao** [goodluck1982/SpaceGroupIrep#26] [General]
IMO, the BC book (“The mathematical theory of symmetry in solids” by C. J. Bradley & A. P. Cracknell) referenced by `SpaceGroupIrep` is rather obscure to read. So, I'm still struggling to figure out the precise definitions of little group and Herring little group used here. Of course, sorry for my superficial knowledge of group theory. 

For the little group and Herring little group, I noticed the following definitions in the book ["Symmetry and Condensed Matter Physics A Computational Approach" by M. El-Batanouny (Boston University) and F. Wooten (University of California, Davis)](https://sg1lib.org/book/1054740/0f0218).

On page 373, the little-group of k, i.e., S_k, is defined as follows: 

![image](https://user-images.githubusercontent.com/11155854/152497154-c3162872-2806-4694-8527-6241c8456ef6.png)

On page 395, it deals with "Herring’s little-group and its Irreps":

![image](https://user-images.githubusercontent.com/11155854/152496519-6753b9dd-4418-4bdd-90da-40d17004c86c.png)

[Here](https://doi.org/10.1016/S0016-0032(42)90462-9) is Conyers Herring's original paper, entitled "Character tables for two space groups", which describes the so-called Herring’s little-group method. 

I'm not sure if these are consistent with the definitions used in `SpaceGroupIrep`. 

BTW, If I have posted some vague or even wrong descriptions of related concepts in the previous issues, please take the discussion started here as a correction.

Regards,
HZ

**Comment by goodluck1982** [comment]
Yes, I also think the BC book is somewhat obscure to read, especially its theoretical part. In fact the concepts of little group and Herring little group are consistent between the BC book and the ref you cited. In my opinion, it's not the best choice to learn the group-theory knowledge from the very beginning through the BC book. It's better that learning group-theory knowledge from a textbook and using the BC book as a reference book. Taking myself for example, the main purpose that I read the theoretical part in the BC book is to make clear the convention it uses. In fact, most of my group-theory concepts are learned from Chinese textbooks and I think the Ruibao Tao's book is a good choice.
![图片](https://user-images.githubusercontent.com/48718961/152510269-a25185f1-694b-441c-871e-e57ff1f25658.png)

**Comment by hongyi-zhao** [comment]
> In fact the concepts of little group and Herring little group are consistent between the BC book and the ref you cited.

Thank you for your confirmation. I also noticed another import note on page 374 in the book by [El-Batanouny (Boston University) and F. Wooten (University of California, Davis)](https://sg1lib.org/book/1054740/0f0218), which I want to share with you:

![image](https://user-images.githubusercontent.com/11155854/152537868-e26184dc-77ec-402c-b006-34f16a6faf4b.png)

So, the little group plays an import role in identifying the degeneracy and degeneracy lifting, for example, due to symmetry breaking caused by some reason. This is a very important method to find or create new quasi-particles in topological condensed matter.

> Taking myself for example, the main purpose that I read the theoretical part in the BC book is to make clear the convention it uses. In fact, most of my group-theory concepts are learned from Chinese textbooks and I think the Ruibao Tao's book is a good choice.

Thank you for sharing your experience with me and telling me the book by Ruibao Tao. I also want to show you the methods I have used or may intend to implement when beginning to enter a new field of knowledge, taking group-theory as an example.

First, I collect as many authoritative books on the subject as I can, in electronic or paper versions. For now, I've collected the following books on group-theory on my machine, among which the book by Ruibao Tao was added according to your recommendation above:

```
$ find . -type f |egrep  '(pdf|djvu)$' 
./Group Theory and Quantum Mechanics by Michael Tinkham (z-lib.org).djvu
./character-tables/10-2.pdf
./Elements-of-Group-Theory-for-Physicists-by-A.W.-Joshi/物理学中的群论基础.pdf
./Elements-of-Group-Theory-for-Physicists-by-A.W.-Joshi/Elements of Group Theory for Physicists by A.W. Joshi (z-lib.org).pdf
./The Mathematical Theory of Symmetry in Solids Representation Theory for Point Groups and Space Groups by Christopher Bradley, Arthur Cracknell (z-lib.org).pdf
./Group-Theory-Application-to-the-Physics-of-Condensed-Matter-by-Mildred-S.-Dresselhaus/Group Theory Application to the Physics of Condensed Matter by Mildred S. Dresselhaus, Gene Dresselhaus, Ado Jorio (z-lib.org).pdf
./Group-Theory-Application-to-the-Physics-of-Condensed-Matter-by-Mildred-S.-Dresselhaus/group-full.pdf
./群论及其在物理学中的应用-谢希德.pdf
./Evarestov-Smirnov1997_Book_SiteSymmetryInCrystals.pdf
./群论在化学中的应用_科顿-无缺页版.pdf
./Group theoretical methods and applications to molecules and crystals by Shoon K. Kim (z-lib.org).pdf
./Group_Theory_SM_Barnes_2010.pdf
./PHYSICS - 群论及其在固体物理中的应用【徐婉棠 & 喀兴林】.pdf
./Heine-GroupTheoryInQuantumMechanics_Volker-Heine.pdf
./group_representation_theory_physicists.djvu
./group_theory_SM.pdf
./Group theory in a nutshell for physicists by Zee, A (z-lib.org).pdf
./Group Representation Theory for Physicists by Jin-Quan Chen, Jialun Ping, Fan Wang (z-lib.org).djvu
./Mulliken-symbols/1.1740655.pdf
./Mulliken-symbols/1.1742716.pdf
./Mulliken-symbols/Mulliken Symbols for Irreducible Representations.pdf
./Symmetry-and-Condensed-Matter-Physics-A-Computational-Approach/Symmetry-and-Condensed-Matter-Physics-A-Computational-Approach.pdf
./Symmetry-and-Condensed-Matter-Physics-A-Computational-Approach/solutionmanual.pdf
./应用群论导引(张端明).pdf
./物理学中的群论-陶瑞宝/物理学中的群论(下册).陶瑞宝.pdf
./物理学中的群论-陶瑞宝/物理学中的群论(上册).陶瑞宝.pdf
./物理学中的群论-陶瑞宝/PHYSICS - 物理学中的群论【陶瑞宝】.pdf
./群论及其在物理和化学中的应用-方可.pdf
./物质结构导论-李俊清-影印版/Chapter4.pdf
./物质结构导论-李俊清-影印版/物质结构导论-李俊清-影印版.pdf
./群论及其在物理中的应用-马中骐.pdf
```

Then I chose some of the books that fit my current interests as my main reference. When I encounter problems that need to be solved, I try to find relevant explanations for a specific problem in all books at the same time, compare and verify the relevant explanations found to obtain a more comprehensive and deeper understanding. In this process, some advanced filtering tools, such as [ripgrep-all](https://github.com/phiresky/ripgrep-all),  also can be used to improve the learning efficiency, or search google books, say, by [this](https://www.google.com/search?tbm=bks&q=%22herring+group%22) or [this](https://www.google.com/search?tbm=bks&q=%22herring+little+group%22).

**Comment by goodluck1982** [comment]
Thank you for sharing your experience too.
